lib/evaluators/usurped.js

```diff
@@ -2,7 +2,7 @@
 
 const _ = require('lodash');
```

Comment:
```suggestion
```

Reply:
Don't know how I missed them, but I've removed the remaining lodash imports.